corehq/messaging/scheduling/views.py

```diff
@@ -309,18 +308,25 @@ def get_scheduled_ajax_response(self):
         )
         total_records = query.count()
         query = query.select_related('schedule')
-        broadcasts = query[self.display_start:self.display_start + self.display_length]
-
-        data = []
-        for broadcast in broadcasts:
-            data.append({
-                'name': broadcast.name,
-                'last_sent': self._format_time(broadcast.last_sent_timestamp),
-                'active': broadcast.schedule.active,
-                'editable': self.can_use_inbound_sms or not broadcast.schedule.memoized_uses_sms_survey,
-                'id': broadcast.id,
-            })
-        return self.datatables_ajax_response(data, total_records)
+        limit = int(self.request.GET.get('limit') or 10)
```

Comment:
To match  the format used below:

```suggestion
        limit = int(self.request.GET.get('limit', 10))
```